### ISSUE TITLE
Pull requests

### ISSUE THREAD

**Issue by diemax** [danitinez/dd-windy-weather#1]
Cada vez que pusheamos lo hacemos a una rama previamente creada para la feature que estamos codeando y de hacemos pull request no? Si pusheamos directo se puede armar alto bardo.

```shell
$ git checkout -b feature-a-codear
$ git push origin  -b feature-a-codear
````

😈

**Comment by danitinez** [comment]
todo a master, a lo macho :D 💃  🗡  nha, xacto como decís... veo si más tarde puedo meter la base del proyecto iOS

**Comment by diemax** [comment]
https://github.com/matteocrippa/awesome-swift

https://github.com/cjwirth/awesome-ios-ui

https://github.com/vsouza/awesome-ios

### Ahi esta la posta parece

**Comment by danitinez** [comment]
Sí, estoy analizando alguna arquitectura piola, estaba viendo VIPER o Clean-Swift...

**Comment by diemax** [comment]
mira que bueno lo que hizo este loco:
https://github.com/baires/scrappy
Para checkear los precios de las compus en Argentina ahora que deberian bajar en teoria

**Comment by danitinez** [comment]
piolazo, le faltaria un frontend y mas sources... pero esta buena la idea

**Comment by diemax** [comment]
Si, estaria para ponerlo online, lo mismo para las ofertas de laburo de NZ

**Comment by diemax** [comment]
Cual API para el clima es la que va? :cloud:

**Comment by danitinez** [comment]
https://openweathermap.org -> api de los pobres, gratarola pero limitada
http://apidev.accuweather.com/developers/forecasts -> esta está piola pero
después de 6 meses es paga

hay un par más , podríamos arrancar con la primera, y cambiar a alguna otra
si hace falta. Quiero mostrar Tº, viento, nubes y mm de lluvia cada 3
horas, como http://www.windguru.com

Ese sitio está bueno, lo usa bastante gente, porque condensa en una sola
pantalla mucha info.

2017-02-28 10:09 GMT+13:00 Diego Maximiliano <notifications@github.com>:

**Comment by diemax** [comment]
![weather](https://cloud.githubusercontent.com/assets/2607767/23381612/f979c556-fda3-11e6-8f23-8ac022750535.png)

Esta copada la data que tira.

**Comment by danitinez** [comment]
Mientras tenga: Tº, viento, lluvia y nubosidad, en intervalos de 3hs creo
que estamos ok

2017-02-28 10:53 GMT+13:00 Diego Maximiliano <notifications@github.com>:
